### ISSUE TITLE
Config crashes due to AladdinConnectGarageDoorOpener

### ISSUE THREAD

**Issue by dfranssen** [apexad/homebridge-garagedoor-command#22]
Setup via admin webapp of HomeBridge results in crash of HomeBridge:
```
[5/4/2020, 17:13:47] [HB Supervisor] Starting Homebridge with extra flags: -I
[5/4/2020, 17:13:47] [HB Supervisor] Started Homebridge v1.0.4 with PID: 5600
[5/4/2020, 17:13:48] Loaded config.json with 1 accessories and 1 platforms.
[5/4/2020, 17:13:48] ---
[5/4/2020, 17:13:49] Loaded plugin: homebridge-config-ui-x
[5/4/2020, 17:13:49] Registering platform 'homebridge-config-ui-x.config'
[5/4/2020, 17:13:49] ---
[5/4/2020, 17:13:49] Loaded plugin: homebridge-garagedoor-command
[5/4/2020, 17:13:49] Registering accessory 'homebridge-garagedoor-command.GarageCommand'
[5/4/2020, 17:13:49] ---
[5/4/2020, 17:13:49] Loading 1 platforms...
[5/4/2020, 17:13:49] [Config] Initializing config platform...
[5/4/2020, 17:13:49] [Config] Running in Service Mode
[5/4/2020, 17:13:49] Loading 1 accessories...
[5/4/2020, 17:13:49] Error loading accessory requested in your config.json at position 1
[5/4/2020, 17:13:50] Error: The requested accessory 'AladdinConnectGarageDoorOpener' was not registered by any plugin.
    at PluginManager.getPluginForAccessory (/usr/lib/node_modules/homebridge/src/pluginManager.ts:197:15)
    at /usr/lib/node_modules/homebridge/src/server.ts:346:37
    at Array.forEach (<anonymous>)
    at Server._loadAccessories (/usr/lib/node_modules/homebridge/src/server.ts:329:29)
    at Server.start (/usr/lib/node_modules/homebridge/src/server.ts:155:12)
    at cli (/usr/lib/node_modules/homebridge/src/cli.ts:78:10)
    at Object.<anonymous> (/usr/lib/node_modules/homebridge/bin/homebridge:17:22)
    at Module._compile (internal/modules/cjs/loader.js:1133:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1153:10)
    at Module.load (internal/modules/cjs/loader.js:977:32)
[5/4/2020, 17:13:50] Got SIGTERM, shutting down Homebridge...
```

**Comment by apexad** [comment]
This is `GarageCommand`. You are likely wanting to use:
https://www.npmjs.com/package/homebridge-aladdin-connect-garage-door

**Comment by dfranssen** [comment]
Are you sure this hasn't been forked ;-) Cfr [config.schema.json](https://github.com/apexad/homebridge-garagedoor-command/blob/62a0d11aea3090f2ddadc25d010e70628718a5bf/config.schema.json) : `"pluginAlias": "AladdinConnectGarageDoorOpener"`

**Comment by apexad** [comment]
Good catch! Will fix this in a new version in a moment.

**Comment by apexad** [comment]
Should be fixed in 0.4.0

**Comment by apexad** [comment]
I wrote the config.schema.jdon got bothbplugins and apparently overlooked this important detail to update the pluginAlias.
